web_widget_html_markdown/demo/bootstrap_markdown.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_groups_form_inherited" model="ir.ui.view">
+        <field name="model">res.groups</field>
+        <field name="inherit_id" ref="base.view_groups_form" />
+        <field name="arch" type="xml">
+            <field name="comment" position="attributes">
+                <attribute name="widget">bootstrap_markdown</attribute>
+            </field>
+        </field>
+    </record>
+</odoo>
```

Comment:
We should modify this, `bootstrap_markdown` widget is used by `web_widget_text_markdown`. Instead as demo data, we can pick any HTML field that is in base Odoo by default and add our HTML markdown widget to it. Maybe the one used in Email templates, or chatter messages. Or I think you found another one somewhere.

Reply:
Gave the widget right name. 
This demo was included for testing purposes. will delete when works perfectly.